docs/application/dotnet/guides/nui/layouts.md

```diff
@@ -0,0 +1,249 @@
+
+### Set View Layout
+
+To layout the children in a `View` as desired, you must create a layout and set it to the `View`. NUI Layouts support the extra settings. For example, you can set the number of columns for a `GridLayout`.
+
+The following code shows how to create `View` and `LinearLayout`, and how to set the layout on `View`.
+`LinearLayout` can be vertical or horizontal. In this example, the linear layout has a horizontal orientation:
+
+```csharp
+View parentView = new View();
+
+var layout = new LinearLayout();
+layout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+
+parentView.Layout = layout;
+```
+
+A `View` can be assigned to a new layout. This change rearranges the children in the View as described by the new layout.
+At a time, you can assign only one layout to a single `View`. If you assign the same layout to multiple Views then the layout is applied only to the last `View`.
+If a layout is moved from one `View` to another, a default layout is assigned to `View`.
+
+The following code shows how to change the layout from `LinearLayout` to `GridLayout` on an event occurrence:
+
+```csharp
+View parentView = new View();
+
+var layoutLinear = new LinearLayout();
+layout.LinearOrientation = LinearLayout.Orientation.Horizontal;
+
+parentView.Layout = layoutLinear;
+
+...
+
+someAction.Event += (sender, e) =>
+{
+  parentView.Layout = new GridLayout();
+}
+```
+
+### Size Specifications
+
+Size specification is used to specify the preferred size of `View` within the layout framework. Height and width are individual values, so each value can have a different specification.
+
+Following are the two size specifications available for `View`:
+
+- Setting **explicit value**
+- **Layout parameters**
+
+* You can provide an explicit value, either using the existing [Size](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Size.html) API or setting a pixel value to the specification.
+You can specify the exact measurement for the width and height value.
+
+  Using [Specification](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_HeightSpecification) API
+
+  ![Specification](./media/size_120x120.png)
+
+  ```csharp
+  View child1 = new View();
+  child1.WidthSpecification = 90;
+  child1.HeightSpecification = 90;
+
+  View child2 = new View();
+  child2.WidthSpecification = 120;
+  child2.HeightSpecification = 120;
+
+  View child3 = new View();
+  child3.WidthSpecification = 90;
+  child3.HeightSpecification = 90;
+  ```
+
+* You can use the layout parameters `MatchParent` or `WrapContent` to set the width and height values as relative sizes. `MatchParent` extends the size of a child to its parent `View`. `WrapContent` shrinks the size of the parent or child `View` to the size of the child `View`.
+
+  * `WrapContent`
+
+    : If the height or width specification of `View` is set to `WrapContent`, that size of the `View` changes to wrap around the child.
```

Comment:
```suggestion
- `WrapContent`: If the height or width specification of `View` is set to `WrapContent`, that size of the `View` changes to wrap around the child.
```

Please use the recommended Markdown format for bullet list: https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#nested-list-spacing

Same comment for Line no. 109 - 111

Reply:
@rishu-samsung or, below form would be also ok. What do you think?

 - `WrapContent`

   If the height or width specification of `View` is set to `WrapContent`, that size of the `View` changes to wrap around the child.

   ...

 - `MatchParent`

   If the height or width specification of `View` is set to `MatchParent`, the height or width extend and fit to parent size.

   ...

Reply:
@mijin85cho 
Yes, this format is also fine.
@Seoyeon2Kim 
Kindly update as @mijin85cho suggested.

Reply:
Okay, could you check the format again?